azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -159,12 +163,11 @@
   //queuedThreadPool is mainly used to monitor jetty threadpool.
   private QueuedThreadPool queuedThreadPool;
 
-  private UserManager userManager;
-  private ProjectManager projectManager;
-  // private ExecutorManagerAdapter executorManager;
-  private ExecutorManager executorManager;
-  private ScheduleManager scheduleManager;
-  private TriggerManager triggerManager;
+  private final UserManager userManager;
+  private final ProjectManager projectManager;
+  private final ExecutorManager executorManager;
+  private final ScheduleManager scheduleManager;
+  private final TriggerManager triggerManager;
   private Map<String, Alerter> alerters;
```

Comment:
added final to the remaining ones. Although this is not a change about refactoring `AzkabanWebServer`. Thus, am not keen on making too many changes there.